site/docs/components/stepped-tracker/examples.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+<LivePreviewControls>
+
+    <LivePreview componentName="stepped-tracker" exampleName="Basic" >
+
+      ### Basic
+
+      The SteppedTracker contains multiple TrackerStep child components. Each TrackerStep indicates its current status through its icon, and the connectors indicate current active step i.e., progress through the process. You should add a label to the TrackerStep using the StepLabel component as follows:
+
+    </LivePreview>
+
+    <LivePreview componentName="stepped-tracker" exampleName="StepProgression" >
+
+      ### Step Progression
+
+      In normal circumstances, once a step is completed you should both advance the active step, and change the status of the current active step in unison.
+
+    </LivePreview>
+
+    <LivePreview componentName="stepped-tracker" exampleName="NonSequentialProgress" >
+
+      ### Non-Sequential Progress
+
+      It may however be appropriate to control the state of steps, and the active step independantly if users are able to revisit previous steps or complete step non-sequentially.
```

Comment:
```suggestion
      It may however be appropriate to control the state of steps, and the active step independently if users are able to revisit previous steps or complete step non-sequentially.
```